roles/keystone/tasks/domains.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: keystone domain dir
+  file: dest=/etc/keystone/domains state=directory
+
+- name: keystone multi domain conf files
+  template: src="etc/keystone/domains/keystone.{{ item }}.conf"
+            dest="/etc/keystone/domains/keystone.{{ item }}.conf"
+  with_items: keystone.multi_domain.domain_conf_files
+  when: keystone.multi_domain.domain_conf_files.0 is defined
```

Comment:
This one is going to be awkward to manage. We don't want to put the details directly in the file, and we don't want to put the file in openstack-envs as is. I think we should just narrow the scope of domain specific auth to be LDAP only, create a template file that takes in the requisite values from env config (or junk role defaults).

This would allow us to just define credential details in the openstack-envs entry.

Does that make sense?